### PR TITLE
[T162438] change membership application confirmation mail to reflect payment type

### PR DESCRIPTION
**Pull request by KaiNissen**
This PR is required by wmde/FundraisingFrontend#911. It is part of [phab:T162438](https://phabricator.wikimedia.org/T162438) and adds the changes to the membership application confirmation mail templates as defined in [wmde/fundraising#1083](https://github.com/wmde/fundraising/issues/1083#issuecomment-289787003).

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
+1

File: i18n/de_DE/mail/membership_application_confirmation/intro.twig [hunk 1]
```diff
@@ -0,0 +1,5 @@
+herzlich willkommen bei Wikimedia Deutschland e.V.
```
Comment (gbirke) [inline i18n/de_DE/mail/membership_application_confirmation/intro.twig:1]: Shouldn't that be 'Herzlich Willkommen'?

Comment (wiese) [reply i18n/de_DE/mail/membership_application_confirmation/intro.twig:1]: Comes after a "Sehr geehrtes Mitglied,\n" or the like so should be correct the way it is (cp. [before](https://github.com/wmde/fundraising-frontend-content/blob/test/i18n/de_DE/mail/membership_application_confirmation/body.twig#L1)). Anyways, this is user-editable content.

**Comment by wiese** [comment]
No problem yet, but we are getting dangerously close (25 char) to the character limit (27 char) for `item_name_donation` used in Sofort's `reason` (see https://www.sofort.com/integrationCenter-eng-DE/content/view/full/2513#h5-1-2)

**Review by wiese — APPROVED** [review]

**Pull request merged**